### ISSUE TITLE
Screen reader doesn't read "Check"

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/vector-addition#426]
For https://github.com/phetsims/qa/issues/1332 ...

In https://github.com/phetsims/vector-addition/issues/424#issuecomment-3606931873, @vperezcadavid99 reported:

> In addition to this, I just realized that the screen reader doesn't read "Check"

<img width="680" height="68" alt="Image" src="https://github.com/user-attachments/assets/c2f03522-5fa9-4406-a7c2-00027ba47198" />

Seen on Windows 25H2 Firefox 145.0.2+NVDA

**Comment by vperezcadavid99** [comment]
Looks good on main, thanks!
Closing
